tests/integration/targets/meraki_ms_access_policies/tasks/main.yml

```diff
@@ -0,0 +1,262 @@
+# Test code for the Meraki Organization module
+# Copyright: (c) 2022, Marcin Woźniak (@y0rune) <y0rune@aol.com>
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test an API key is provided
+  ansible.builtin.fail:
+    msg: Please define an API key
+  when: auth_key is not defined
+
+- name: Query all access polices
+  cisco.meraki.meraki_ms_access_policies:
+    auth_key: "{{ auth_key }}"
+    state: query
+    org_name: "{{ test_org_name }}"
+    net_name: "{{ test_net_name }}"
+  delegate_to: localhost
+  register: query_all
+
+- name: Create access policy with auth_method is "Meraki authentication"
+  cisco.meraki.meraki_ms_access_policies:
+    auth_key: "{{ auth_key }}"
+    state: present
+    name: "Meraki authentication policy"
+    auth_method: "Meraki authentication"
+    net_id: "{{ test_net_id }}"
```

Comment:
Please change it to be similar to the `org_name` and `test_switch_net_name` for all the tests. I have separate networks for each device type because I need to develop for MR, MS, and MX devices. Since this is MS, it should be `test_switch_net_name`. Same should apply for all tests in this one.